src/unitxt/operators.py

```diff
@@ -1743,6 +1809,61 @@ def process(self, stream: Stream, stream_name: Optional[str] = None) -> Generato
         yield from stream
 
 
+class JoinStreams(MultiStreamOperator):
+    """Merges multiple streams into a single stream.
```

Comment:
Need to update the doc - and most important add instructions.

Reply:
wdym instructions? I followed the example of the merge operator